src/components/MoviesList/MoviesList.jsx

```diff
@@ -0,0 +1,16 @@
+import { MovieCard } from '../MovieCard';
+import './MoviesList.scss';
+
+export const MoviesList = ({ movies }) => (
+  <div className="movies">
+    {movies && movies.map(movie => (
```

Comment:
movies it is array and it is always true, if you want to check if something exist in array use length